.github/workflows/welcome-first-time-contrib.yml

```diff
@@ -1,34 +1,30 @@
 #This action is centrally managed in https://github.com/asyncapi/.github/
 #Don't make changes to this file in this repo as they will be overwritten with changes made to the same file in above mentioned repo
 
-#########
-#disabled because of https://github.com/asyncapi/.github/issues/73
-#########
+name: Welcome first time contributors
```

Comment:
must be disabled, please replace with https://github.com/asyncapi/.github/blob/master/.github/workflows/welcome-first-time-contrib.yml

Reply:
ok will do